### PR TITLE
Buffs Mechs to be more relevant with current weapon damage values

### PR DESCRIPTION
**Pull request by githubuser4141**
This simply buffs Mech's health and armor values so they're not completely worthless in combat. I thought this was worth putting out here as the standard Lasgun which everyone carries can currently put down a Gygax 10-15 hits which is fairly 'meh' especially at higher population.

This shouldn't be OP; Mechs still require a decent amount of effort in RnD/Materials, have limited power and are clunky and slow enough to be overwhelmed if they're out alone. Also increases the equipment slots of the Odysseus and Ripley.

tl;dr version: Assuming we're using basic 40 damage lasguns, 

buffs the Durand from ~15 hits to kill (HtK) to about ~20 HtK

buffs the Gygax from ~12 HtK to about ~15 HtK

buffs the Phazon (Probably can't be built at this point) from ~12 Htk to around ~15ish. 

buffs the Ripley, Firefighter and Odysseus so they don't crumple from two or three rounds. You'll still likely lose to a naked guy with a lasgun, however.

Shouldn't be any bugs as I've only changed health/resistance values unless I mistyped something.

### CODE REVIEW

**Comment by Daelso** [comment]
Looks fine to me - I'm not sure the place mechs will have but w/e
